Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/overview/items/PersonCertificateCard.kt

```diff
@@ -25,32 +25,34 @@ class PersonCertificateCard(parent: ViewGroup) :
         PersonOverviewItemBinding.inflate(layoutInflater, itemView.findViewById(R.id.card_container), true)
     }
 
+    // TODO: Update curItem with the right certificate
     override val onBindData: PersonOverviewItemBinding.(
         item: Item,
         payloads: List<Any>
     ) -> Unit = { item, payloads ->
         val curItem = payloads.filterIsInstance<Item>().lastOrNull() ?: item
 
         val color = when {
-            curItem.certificate.isValid -> curItem.colorShade
+            curItem.admissionState.primaryCertificate!!.isValid -> curItem.colorShade
```

Comment:
`!!` is not required

Reply:
This is not final, it's just until my PR will be merged into this one #4491